packages/fxa-settings/src/lib/config.ts

```diff
@@ -1,17 +1,107 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { deepMerge } from './utilities';
+
+export const META_CONFIG = 'fxa-config';
+
 export interface Config {
   env: string;
+  version: string;
   sentry: {
     url: string;
     dsn: string;
   };
-  version: string | undefined;
+  servers: {
+    gql: {
+      url: string;
+    };
+    // NOTE: due to the nature and specificity of the requests we need to
+    // talk directly to the auth server for the following routes:
+    // - POST /password/change/start
+    // - POST /password/change/finish
```

Comment:
Delete account as well.

Maybe this comment is better in `App.tsx` until we get it configured.